src/Model/ElementForm.php

```diff
@@ -9,6 +9,10 @@
 use DNADesign\ElementalUserForms\Control\ElementFormController;
 use SilverStripe\Control\RequestHandler;
 
+/**
+ * @method SilverStripe\ORM\HasManyList<EmailRecipient> EmailRecipients()
+ * @method SilverStripe\ORM\HasManyList<SubmittedForm> Submissions()
```

Comment:
These are added through a trait.
Yes, they should appear here - but we need to either import the `EmailRecipient` and `SubmittedForm` with `use` statements, or use the FQCN here.

Reply:
Having updated the docblock writer, using the FCQN here